### PR TITLE
FIX broken re-runs of the install script on k8s 1.18.x

### PR DESCRIPTION
**Pull request by mzaneri**

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -39,6 +39,9 @@ function init() {
         API_SERVICE_ADDRESS="$LOAD_BALANCER_ADDRESS:$LOAD_BALANCER_PORT"
     fi
 
+    #This is needed on k8s 1.18.x as $PRIVATE_ADDRESS is found to have a newline
+    PRIVATE_ADDRESS=$(echo "$PRIVATE_ADDRESS" | tr '\n' ' ')
```
Comment (areed) [inline scripts/install.sh:43]: why replace with a space instead of the empty string?

Comment (mzaneri) [reply scripts/install.sh:43]: tr: when not truncating set1, string2 must be non-empty

Comment (mzaneri) [reply scripts/install.sh:43]: maybe I should also strip leading whitespace after too?

Comment (areed) [reply scripts/install.sh:43]: yes all whitespace should be stripped

**Pull request merged**